frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -0,0 +1,98 @@
+<template lang='pug'>
+  proposal-template(
+    :title='L("Add new members")'
+    :rule='{ value: 5, total: 10 }'
+    :disabled='$v.form.$invalid || ($v.steps[config.steps[currentStep]] && $v.steps[config.steps[currentStep]].$invalid)'
+    :maxSteps='config.steps.length'
+    :currentStep.sync='currentStep'
+    @submit='submit'
+  )
+    // | TBD in another task
+    // | https://github.com/okTurtles/group-income-simple/issues/609
+    div(v-if='currentStep === 0' key='0')
+      label.field
+        i18n.label Username
+        .input-combo
+          input.input(
+            type='text'
+            name='mincome'
+            required=''
+            :class="{ 'error': $v.form.member.$error }"
+            v-model='form.member'
+            @keyup.enter='next'
+          )
+      button.link.c-addMember(
+        @click="addNewMember"
+      )
+        i.icon-plus
+        i18n Add more people
+      // invite
+</template>
+
+<script>
+import Invite from '@pages/Invite.vue'
+import { validationMixin } from 'vuelidate'
+import { required } from 'vuelidate/lib/validators'
+import ProposalTemplate from './ProposalTemplate.vue'
+
+export default {
+  name: 'AddMembers',
+  components: {
+    Invite,
+    ProposalTemplate
+  },
+  mixins: [
+    validationMixin
+  ],
+  data () {
+    return {
+      currentStep: 0,
+      v: { type: Object },
```

Comment:
Hmm, I see you changed it below (except for the `v` thing), but not here. Can you move the `currentStep` and `v` to ephemeral? What even is `v`, it doesn't seem to be used? If it's not used it should be removed.

Reply:
as I said, I didn't move it because this file is just a placeholder without any real work on it yet. but changed now